### PR TITLE
Doc rework (user guide, part 2)

### PR DESCRIPTION
**Pull request by stephenfin**
Continue building up the user guide by adding a tutorial document. This is based off a cleaned up tutorial document.

You should view this on a commit-by-commit basis to best understand the individual changes.

### CODE REVIEW

File: doc/usage/quickstart.rst [hunk 1]
```diff
@@ -145,6 +149,8 @@ an argument to see which targets are available.
    toolchain for you.
 
 
+.. todo:: Move this whole section into a guide on rST or directives
```
Comment (stephenfin) [inline doc/usage/quickstart.rst:152]: I have all these `todo`\'s resolved in follow-up patches. I've submitted this separately to keep things easy to review

File: doc/usage/quickstart.rst [hunk 2]
```diff
@@ -300,15 +318,14 @@ features of intersphinx.
 More topics to be covered
 -------------------------
 
-- :doc:`Other extensions <extensions>`:
-
```
Comment (tk0miya) [inline doc/usage/quickstart.rst]: Did you check the rendered page? I think this empty line is needed.

File: doc/usage/quickstart.rst [hunk 3]
```diff
@@ -320,6 +337,6 @@ More topics to be covered
        another directory, the :term:`configuration directory`.  Refer to the
        :program:`sphinx-build man page <sphinx-build>` for more information.
 
-.. |more| image:: more.png
+.. |more| image:: ../_static/more.png
```
Comment (tk0miya) [inline doc/usage/quickstart.rst]: You can use `/_static/more.png` instead. Sphinx considers a absolute path as a relative path from srcdir.

File: doc/usage/quickstart.rst (hunk 1 above)
Comment (tk0miya) [reply doc/usage/quickstart.rst:152]: Reasonable, but this todo is useful only for us. What about comment it out?

**Review by tk0miya — CHANGES_REQUESTED** [review]
LGTM with nits

File: doc/usage/quickstart.rst (hunk 1 above)
Comment (stephenfin) [reply doc/usage/quickstart.rst:152]: I don't think it matters. I'll be fixing this literally within days. Can we just keep this as is?

File: doc/usage/quickstart.rst (hunk 2 above)
Comment (stephenfin) [reply doc/usage/quickstart.rst]: Done

File: doc/usage/quickstart.rst (hunk 3 above)
Comment (stephenfin) [reply doc/usage/quickstart.rst]: Done

**Comment by stephenfin** [comment]
This is done now. As noted, the TODOs will be removed very soon. I think it's important that they're left there just in case I disappear of the face of the earth or something, heh

**Comment by tk0miya** [comment]
I said to replace TODOs by comments. Why do you want to show TODO for users?

**Comment by stephenfin** [comment]
> I said to replace TODOs by comments. Why do you want to show TODO for users?

So users will know this is a WIP. They will only be there for a few days. In addition, I don't want to have to rework the ~20 patches that I have in the series after these (I'm submitting these separately).

**Comment by tk0miya** [comment]
I don't like showing TODO block for users. But it says "time flies like an arrow". So let's go with as is.

**Comment by tk0miya** [comment]
Merged. Thanks,

**Comment by stephenfin** [comment]
Yay. Thanks! :+1:

**Pull request merged**